src/helpers/locale.ts

```diff
@@ -106,3 +113,66 @@ function localizeLiteral(content: string, locale: Locale): string {
   });
   return localized;
 }
+
+export function canonicalizeCFRule(
+  cf: ConditionalFormatRule,
+  locale: Locale
+): ConditionalFormatRule {
+  return changeCFRuleLocale(cf, (content) => canonicalizeContent(content, locale));
+}
+
+export function localizeCFRule(cf: ConditionalFormatRule, locale: Locale): ConditionalFormatRule {
+  return changeCFRuleLocale(cf, (content) => localizeContent(content, locale));
+}
+
+function changeCFRuleLocale(
+  rule: ConditionalFormatRule,
+  changeContentLocale: (content: string) => string
+) {
+  rule = deepCopy(rule);
+  switch (rule.type) {
+    case "CellIsRule":
+      // Only change value for number operators
+      if (
+        [
+          "Between",
+          "Equal",
+          "GreaterThan",
+          "GreaterThanOrEqual",
+          "LessThan",
+          "LessThanOrEqual",
+          "NotBetween",
+          "NotEqual",
+        ].includes(rule.operator)
+      ) {
+        rule.values = rule.values.map((v) => changeContentLocale(v));
+      }
```

Comment:
I would use a switch/case statement, explicitly handle every operator (including for non-number operators) and return in each handled case.
With the current implementation, if we add a new operator, there's a high chance the dev will forget to adapt this.
With every case explicitly checked, and each of them returning the value, Typescript will warn it if there's an un-handled case.